Cargo.toml says indexmap 1.0.0 is requires, but indexmap 1.2.0 is actually required

```
error[E0599]: no method named `insert_full` found for type `indexmap::set::IndexSet<&'a [u8]>` in the current scope
  --> /Users/bjorn/.cargo/registry/src/github.com-1ecc6299db9ec823/object-0.14.0/src/write/string.rs:22:31
   |
22 |         let id = self.strings.insert_full(string).0;
   |
```

Updating `Cargo.lock` from `indexmap 1.0.1` to `indexmap 1.2.0` fixed the problem.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 authors = ["Nick Fitzgerald <fitzgen@gmail.com>", "Philip Craig <philipjcraig@gmail.com>"]
 name = "object"
-version = "0.14.0"
+version = "0.14.1"
 edition = "2018"
 description = "A unified interface for parsing object file formats."
 keywords = ["object", "loader", "elf", "mach-o", "pe"]

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,8 +17,8 @@ scroll = { version = "0.9", default-features = false }
 target-lexicon = { version = "0.8" }
 uuid = { version = "0.7", default-features = false }
 flate2 = { version = "1", optional = true }
-crc32fast = { version = "1", optional = true }
-indexmap = { version = "1", optional = true }
+crc32fast = { version = "1.2", optional = true }
+indexmap = { version = "1.1", optional = true }
 
 [dependencies.goblin]
 version = "0.0.24"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -9,6 +9,9 @@ license = "Apache-2.0/MIT"
 repository = "https://github.com/gimli-rs/object"
 exclude = ["/.coveralls.yml", "/.travis.yml"]
 
+[package.metadata.docs.rs]
+all-features = true
+
 [dependencies]
 scroll = { version = "0.9", default-features = false }
 target-lexicon = { version = "0.8" }

